package.json

```
@@ -17,15 +17,15 @@
     "test-watch:e2e": "cross-env PWTEST_WATCH=1 playwright test",
     "lint": "eslint --ext .js,.vue,.ts *.config.* src tests build",
     "fmt": "eslint --ext .js,.vue,.ts *.config.* src tests build --fix",
-    "markdownlint": "markdownlint --ignore node_modules/ --ignore dist/ --ignore dist_electron/ ./",
+    "markdownlint": "markdownlint --ignore node_modules/ --ignore dist/ --ignore dist_electron/ --ignore build/vendored ./",
```

`vendored`には直下にreadmeがあるので、こうのがいいかも
```suggestion
    "markdownlint": "markdownlint --ignore node_modules/ --ignore dist/ --ignore dist_electron/ --ignore build/vendored/voicevox_engine/speaker_info ./",
```